pkgs/by-name/gn/gnome-user-share/package.nix

```diff
@@ -60,6 +60,7 @@ stdenv.mkDerivation (finalAttrs: {
     "-Dmodules_path=${apacheHttpd}/modules"
     "-Dsystemduserunitdir=${placeholder "out"}/etc/systemd/user"
   ];
+  mesonBuildType = "release";
```

Comment:
Hmm, I guess, ideally, cargo would also have `plain` profile and the build script would detect `buildtype=plain` and then call `cargo build --plain`, leaving the rustc optimization-related build flags entirely up to stdenv.

https://github.com/GNOME/gnome-user-share/blob/a31d3ca987ed80e223a4003d49063ffdb91a1b17/src/meson.build#L20

But I suppose, switching to `release` is fine, as there is no C code built by Meson in the project, that would be affected.